### PR TITLE
New Rich Presence: Multporn

### PR DESCRIPTION
**Pull request by SammCheese**
Name: Multporn.net
About: Website for Adult Comics

Working Proof below, although i cant show the Site as its Full of NSFW Content.

image
image
image
image

Ill gladly Fix any Errors, Mistakes and Bugs you encounter!

### CODE REVIEW

File: websites/M/Multporn/presence.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-var presence = new Presence({
+const presence = new Presence({
   clientId: "791258115622305813"
-});
+}),
 
-var browsingStamp = Math.floor(Date.now() / 1000);
-var viewing: any;
+ browsingStamp = Math.floor(Date.now() / 1000);
+let viewing: unknown;
```
Comment (ririxidev) [inline websites/M/Multporn/presence.ts:6]: HTMLElement :)

File: websites/M/Multporn/presence.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
+const presence = new Presence({
+  clientId: "791258115622305813"
+}),
+
```
Comment (ririxidev) [inline websites/M/Multporn/presence.ts:4]: delete this blank line

File: websites/M/Multporn/presence.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      presenceData.startTimestamp = browsingStamp;
+      presenceData.details = "Searching for Comics";
+    }
+  }
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle(); //Clears the tray title for mac users
+    presence.setActivity(); /*Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name*/
+  } else {
+    presence.setActivity(presenceData); //Update the presence with all the values from the presenceData object
```
Comment (ririxidev) [inline websites/M/Multporn/presence.ts:139]: remove these comment lines

File: websites/M/Multporn/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "url": "www.multporn.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/aBu33JP.png",
+  "thumbnail": "https://i.imgur.com/v5BuHwD.png",
+  "color": "#cc0000",
+  "tags": ["nsfw", "multporn"],
+  "category": "other",
+  "regExp": "multporn[.]net",
+  "iFrameRegExp": "REGEXP",
+  "iframe": false
```
Comment (ririxidev) [inline websites/M/Multporn/dist/metadata.json:19]: delete them bc u dont need regexp nor even iframeregexp and iframe

File: websites/M/Multporn/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "Samm-Cheese#9500",
```
Comment (ririxidev) [inline websites/M/Multporn/dist/metadata.json:3]: without tag

**Review by ririxidev — CHANGES_REQUESTED** [review]
and upload new proof of working presence

**Comment by SammCheese** [comment]
New proof of working presence
image
image
image

File: websites/M/Multporn/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  "author": {
+    "name": "Samm-Cheese",
+    "id": "372148345894076416"
+  },
+  "service": "Multporn",
+  "description": {
+    "en": "Multporn.net is an Adult Website for Comics of any kind."
+  },
+  "url": "www.multporn.net",
+  "version": "1.0.3",
```
Comment (Bas950) [inline websites/M/Multporn/dist/metadata.json:11]: 1.0.0 on release

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Multporn/dist/metadata.json (hunk 6 above)
Comment (SammCheese) [reply websites/M/Multporn/dist/metadata.json]: Version Number set back to 1.0.0

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**